blocks/magazine-hero/magazine-hero.js

```diff
@@ -56,13 +56,9 @@ function createLeftOverlay(overlay, config) {
   return leftSide;
 }
 
-function scrollToFirstParagraph() {
-  for (const p of document.querySelectorAll('.default-content-wrapper p')) {
-    if (!p.querySelector('iframe')) {
-      p.scrollIntoView({ behavior: 'smooth' });
-      break;
-    }
-  }
+function scrollToNext(event) {
+  const heroContainer = event.target.closest('.magazine-hero-container');
```

Comment:
This skips the quote on https://162-magazine-explore-button--24life--hlxsites.hlx.page/magazine/volume-3-issue-3

how about 
```js
block.closest('.magazine-hero-wrapper').nextElementSibling || block.closest('. section).nextElementSibling